src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -1091,15 +1042,15 @@ class AddItemsPage extends Component {
               </div>
               <div className="submit-buttons">
                 <button
-                  type="submit"
+                  type="button"
                   disabled={invalid || showOnly}
                   onClick={() => this.previousPage(values, invalid)}
                   className="btn btn-outline-primary btn-form btn-xs"
                 >
                   <Translate id="react.default.button.previous.label" defaultMessage="Previous" />
                 </button>
                 <button
-                  type="submit"
+                  type="button"
```

Comment:
Why do you change those? I just want to be in touch with this change, as it is not first time I see you changing those.
I guess that keeping it as `submit` doesn't just have any sense, since we provide `onClick` and our `onSubmit` does it: `() => {}`
Am I right?

Reply:
that's one part of it but also since I was implementing a on enter behavior on _product select input_ and this component is inside of a  _form_ tag it  would get that enter event and perform a click on button that is type submit